src/mmw/js/src/analyze/views.js

```diff
@@ -16,6 +17,8 @@ var $ = require('jquery'),
     tabContentTmpl = require('./templates/tabContent.html'),
     barChartTmpl = require('../core/templates/barChart.html');
 
+var M_IN_KM = 1000000;
```

Comment:
Actually, M2_IN_KM2 makes more sense. However, there really isn't a good way to represent squaring in a variable name. Maybe just use a comment.